community-containers/mollysocket/mollysocket.json

```
@@ -0,0 +1,33 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-mollysocket",
+            "display_name": "MollySocket",
+            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/mollysocket",
+            "image": "ghcr.io/mollyim/mollysocket",
+            "image_tag": "1",
+            "restart": "always",
+            "ports": [
+                "8020:8020"
+            ],
+            "volumes": [
+                {
+                    "source": "nextcloud_aio_mollysocket",
+                    "destination": "/data",
+                    "writeable": true
+                }
+            ],
+            "working_dir": "/data",
+            "command": "server",
+            "environment": [
+                "TZ=%TIMEZONE%",
+                "MOLLY_DB=\"/data/mollysocket.db\"",
+                "MOLLY_ALLOWED_ENDPOINTS=[\"*\",\"https://push.{$NC_DOMAIN}\"]",
+                "MOLLY_ALLOWED_UUIDS=[\"*\"]",
```

I am not 100% sure about these:
```suggestion
                "MOLLY_DB=/data/mollysocket.db",
                "MOLLY_ALLOWED_ENDPOINTS=[\"*\",\"https://push.%NC_DOMAIN%\"]",
                "MOLLY_ALLOWED_UUIDS=[\"*\"]",
```